packages/cxl-ui/scss/cxl-institute-content.scss

```diff
@@ -0,0 +1,64 @@
+@import "./utils/mixins";
```

Comment:
Follow https://github.com/justintadlock/mythic/tree/master/resources/scss structure.

Reply:
He does have defined break points but using direct `min-width `etc. although we have been using `respond-above(small)` and it also has `respond-between(small, medium)`. I'm currently not seeing the value of his scss structure + https://github.com/justintadlock/mythic/blob/master/resources/scss/settings/_breaks.scss

On another subject, if we would like to convert our styles to mobile-first, what would be the best steps?

Reply:
We can just adopt the structure, directory / file naming for one.

Going with `sass-mq` should handle the rest of actual meat functionality for us.

Min-width-only is better to start with, it keeps out hacky bs. When a real use case arises to the type of "I really can't do this without some `between` type thing", I'd like to see. It may be a conceptual problem that needs to be fixed instead.

Mobile-first requires a bit of a refactor (re-ordering of cascade, and re-assigning mq-s), but we should do it before distributing this framework to any new people.

Reply:
Yes, lets do mobile first refactor asap. Question is, before or after institute merge to master?

Reply:
Nah, I will do it right now